react-charts/stack-chart/step6.md

```diff
@@ -1,54 +1,46 @@
-Now we'll add axis labels to the `Chart` component...
+Add a legend to the stack chart.
 
-## Task
+1) <strong>Make sure the `App.js` file is still open.</strong>
 
-1) Make sure the App.js file is still open
+2) <strong>Locate the code for the `padding` property in the `<Chart>` component.</strong>
 
-2) Locate the code that looks like the following:
+It should look like this:
 
 <pre class="file">
 &lt;Chart
   containerComponent={&lt;ChartVoronoiContainer labels={({ datum }) =&gt; `${datum.name}: ${datum.y}`} /&gt;}
   domainPadding={{ x: [30, 25] }}
+  padding={{
+    bottom: 75,
+    left: 75,
+    right: 50,
+    top: 50
+  }}
   height={250}
   width={600}
 &gt;
 </pre>
 
-3) Add the following property to that section:
+3) <strong>Update the padding for the right value to accommodate the legend.</strong>
 
-<pre class="file" data-target="clipboard">
-padding={{
-  bottom: 75,
-  left: 75,
-  right: 50,
-  top: 50
-}}
-</pre>
-
-- This will add more padding to the bottom and left properties to accommodate the axis labels.
-
-4) Locate the code that looks like the following:
+  `right: 200`
 
-<pre class="file">
-&lt;ChartStack&gt;
-...
-&lt;/ChartStack&gt;
-</pre>
+4) <strong>Add the `legendOrientation` property to the `<Chart>` component.</strong>

```

Comment:
Update this step to mimic how its done in the area chart tutorial:

<img width="769" alt="Screen Shot 2019-12-09 at 9 26 10 AM" src="https://user-images.githubusercontent.com/20118816/70443345-f4d76f80-1a65-11ea-92cf-01031a0d6814.png">